### DISCUSSION TITLE
Can I use skeleton as an UI components library in React?

### DISCUSSION THREAD

**Discussion by jellli** [skeletonlabs/skeleton#1262] [Q&A] [answered]
I really like the design of Skeleton, but I'm a React user :<
Maybe I can use the class name of Skeleton in React, like the usage in [daisyui](https://daisyui.com/?lang=en)?

**Comment by Sarenor** [comment]
That's actually possible for everything in the [elements](https://www.skeleton.dev/elements/core) category!

Mind you, we haven't tested it, won't officially support it and won't recommend it either.
But it's possible!

Or you could:
<img width="361" alt="image" src="https://user-images.githubusercontent.com/31982384/229442400-0e11795f-6524-4fed-b346-47021c46aa7c.png">

**Comment by endigo9740** [comment, marked as answer]
@hoonliu simple answer is we only support Svelte and Svelte components right now. But we have discussed ways to open up Skeleton to additional frameworks in the future. I wouldn't expect it any time soon, unfortunately.
